Partition_2/doc/Partition_2/CGAL/partition_is_valid_2.h

```diff
@@ -35,7 +35,7 @@ with the representation type determined by `std::iterator_traits<InputIterator>:
 \sa `CGAL::greene_approx_convex_partition_2()` 
 \sa `CGAL::optimal_convex_partition_2()` 
 \sa `CGAL::partition_is_valid_2()` 
-\sa `CGAL::is_convex_2()` 
+\sa `CGAL::Is_convex_2()` 
```

Comment:
function or class?  (i.e. mismatch between `I` and `()`)

Reply:
It is a fix for a missing link from [here ](https://doc.cgal.org/latest/Partition_2/group__PkgPartition2Ref.html#gaa4c7b5c06a64608fca50ddb2af49d498)to [here](https://doc.cgal.org/latest/Partition_2/classCGAL_1_1Is__convex__2.html)